packages/server-core/src/user/avatar/avatar.hooks.ts

```diff
@@ -155,6 +155,28 @@ const updateUserAvatars = async (context: HookContext<AvatarService>) => {
   }
 }
 
+const addAndSortByUserName = async (context: HookContext<AvatarService>) => {
+  if (!context.params.query) return
+
+  const userNameAlias = 'userName'
+
+  const sort = context.params.query.$sort?.['owner']
+  delete context.params.query.$sort?.['owner']
+  const nameLike = context.params.query.name as any
+  delete context.params.query.name
+
+  const query = context.service.createQuery(context.params)
```

Comment:
This should only be running if `sort` was there

Reply:
This `query` was added because we want to show user's name instead of userId in the **owner** column of the avatar table.

![image](https://user-images.githubusercontent.com/55396651/275813706-7ab71359-81b3-4e1a-bdcc-ccd1e74cae34.png)